omero_marshal/decode/decoders/plate.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2015 Glencoe Software, Inc. All rights reserved.
+#
+# This software is distributed under the terms described by the LICENCE file
+# you can find at the root of the distribution bundle.
+# If the file is missing please request a copy by contacting
+# jason@glencoesoftware.com.
+#
+
+from ... import SCHEMA_VERSION
+from .annotation import AnnotatableDecoder
+from omero.model import PlateI
+
+
+class Plate201501Decoder(AnnotatableDecoder):
+
+    TYPE = 'http://www.openmicroscopy.org/Schemas/OME/2015-01#Plate'
+
+    OMERO_CLASS = PlateI
+
+    def decode(self, data):
+        v = super(Plate201501Decoder, self).decode(data)
+        self.set_property(v, 'name', data.get('Name'))
+        self.set_property(v, 'description', data.get('Description'))
```

Comment:
As discussed earlier this morning this should be extended to cover the other [Plate attributes](http://www.openmicroscopy.org/Schemas/Documentation/Generated/OME-2016-06/ome_xsd.html#Plate)